python_modules/dagster-graphql/dagster_graphql/schema/pipelines/pipeline.py

```diff
@@ -87,27 +87,73 @@ def parse_time_range_args(args):
     return before_timestamp, after_timestamp
 
 
-class GrapheneMaterializationCountSingleDimension(graphene.ObjectType):
-    materializationCounts = non_null_list(graphene.Int)
+class GrapheneTimePartitionRange(graphene.ObjectType):
+    startTime = graphene.NonNull(graphene.Float)
+    endTime = graphene.NonNull(graphene.Float)
+    startKey = graphene.NonNull(graphene.String)
+    endKey = graphene.NonNull(graphene.String)
 
     class Meta:
-        name = "MaterializationCountSingleDimension"
+        name = "TimePartitionRange"
 
 
-class GrapheneMaterializationCountGroupedByDimension(graphene.ObjectType):
-    materializationCountsGrouped = graphene.NonNull(graphene.List(non_null_list(graphene.Int)))
+class GrapheneTimePartitions(graphene.ObjectType):
+    ranges = non_null_list(GrapheneTimePartitionRange)
 
     class Meta:
-        name = "MaterializationCountGroupedByDimension"
+        name = "TimePartitions"
 
 
-class GraphenePartitionMaterializationCounts(graphene.Union):
+class GrapheneDefaultPartitions(graphene.ObjectType):
+    materializedPartitions = non_null_list(graphene.String)
```

Comment:
I don't think it's worth changing now, but I think in a follow-up it might be nice to make this `materializedKeys` and `unmaterializedKeys` so it's more similar to `startKey` in the other types